src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubAttribute.cs

```diff
@@ -27,5 +28,11 @@ public EventHubAttribute(string eventHubName)
         /// </summary>
         [AutoResolve]
         public string EventHubName { get; private set; }
+
+        /// <summary>
+        /// Optional connection name. If missing, tries to use a registered event hub sender.
+        /// </summary>
+        [AppSetting]
```

Comment:
Ok. But you're saying that the [AppSetting] that I have applied on the base interface actually doesn't do anything and it can be removed with no effect?